components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/callback/OAuthCallbackManagerTest.java

```diff
@@ -0,0 +1,78 @@
+package org.wso2.carbon.identity.oauth.callback;
+
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.application.authentication.framework.model.AuthenticatedUser;
+import org.wso2.carbon.identity.oauth2.IdentityOAuth2Exception;
+
+import static org.mockito.Matchers.any;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertFalse;
+
+/**
+ * Unit tests for OAuthCallbackManager.
+ */
+@PrepareForTest({OAuthCallbackHandlerRegistry.class})
+public class OAuthCallbackManagerTest extends PowerMockTestCase {
+
+    @Mock
+    private OAuthCallbackHandlerRegistry oAuthCallbackHandlerRegistry;
+
+    @Test
+    public void testHandleCallbackNoCallBackHandlerCanbeFound() throws Exception {
+
```

Comment:
Remove new line after method signature

Reply:
We all have agreed to add a new line after the method signature. 
So this is the conventions.